github/github_test.go

```diff
@@ -1054,3 +1054,12 @@ func TestFormatRateReset(t *testing.T) {
 		t.Errorf("Format is wrong. got: %v, want: %v", got, want)
 	}
 }
+
+func TestNamedStructAccessor_noPanic(t *testing.T) {
```

Comment:
Optional suggestion, but `TestNestedStructAccessorNoPanic` might be a better name for the test. There's no need for an underscore, since the text before the underscore doesn't refer to any method or type.

I also added "Nested" in there since that's really the most important part of this test, the fact that _nested_ accessors work without panic. A single accessor wouldn't panic even before this change.

See https://godoc.org/testing#hdr-Examples for where the underscore naming convention comes from.

Reply:
Thanks @shurcooL! I have made the change.

> A single accessor wouldn't panic even before this change.

However, a single accessor for a named struct would give error that the method doesn't exist before this change. That was my idea behind the name. I understand with the convention however.